sapphire/analysis/coincidences.py

```diff
@@ -446,11 +447,70 @@ def _do_search_coincidences(self, timestamps, window):
 
 
 class CoincidencesESD(Coincidences):
-    """Store coincidences differently for the ESD
+    """Store coincidences specifically using the ESD
+
+    This is a subclass of :class:`Coincidences`. In addition to searching for
+    coincidences, this subclass stores the paths to the station_groups that
+    where used to look for coincidences in a lookup-table, and also stores the
+    original station info and event_id for each coincidence.
+
+    Suppose you want to search for coincidences between stations 501 and 503.
+    First, download the data for these stations from the ESD. Suppose you
+    stored the data in the '/s501' and '/s503' groups in the 'data' file.
+    Then::
+
+        >>> station_groups = ['/s501', '/s503']
+        >>> coin = CoincidencesESD(data, '/coincidences', station_groups)
+        >>> coin.search_and_store_coincidences()
+
+    If you want a more manual method, replace the last line with::
 
-    This subclass stores the paths to the station_groups that where
-    used to look for coincidences in a lookup-table, and also
-    stores the original station info and event_id for each coincidence.
+        >>> coin.search_coincidences(window=50000)
+        >>> coin.process_events()
+        >>> coin.store_coincidences()
+
+    You can then provide different parameters to the individual methods.
+    See the corresponding docstrings.
+
+    Once the coincidences are stored, there will be a `coincidences` table in
+    the group. This table has multiple columns used for storing analysis
+    results like shower direction and energy. At this point, they contain no
```

Comment:
The reconstructions modules in sapphire creates a new table for results, these fields are used by the simulations to store the simulation inputs.